_data/gallery.yaml

```
@@ -123,3 +123,10 @@
   ref: 91b8046dfc51e0623700ece1372e64dc770ae71e
   repo_url: https://github.com/sunnysinghnitb/covid-19-world-dashboard
   image_url: https://raw.githubusercontent.com/sunnysinghnitb/covid-19-world-dashboard/master/thumbnail.png
+  
+  - title: IexFinder: Interactive Symbolic Calculation of Extreme Values
```

Looks like we can remove the small indent:

```suggestion
- title: IexFinder: Interactive Symbolic Calculation of Extreme Values
```